### PR TITLE
image-zoomable

### PR DESCRIPTION
**Pull request by adobenitesh**
Jira ID: EXLM-324

Test URLs:

- Before: https://main--exlm--adobe-experience-league.hlx.page/docs/blueprints-learn/architecture/customer-journeys/journey-optimizer/journey-optimizer

- After: https://feature-exlm-324-img--exlm--adobe-experience-league.hlx.live/docs/blueprints-learn/architecture/customer-journeys/journey-optimizer/journey-optimizer

### CODE REVIEW

File: blocks/img/img.js [hunk 1]
```diff
@@ -1,4 +1,73 @@
-export default function decorate(block) {
+import { createTag } from '../../scripts/scripts.js';
+
+function imgZoomable() {
+  const modalImage = Array.from(document?.querySelectorAll('.img.modal-image'));
+
+  // Functions to open and close a modal
```
Comment (thomsebastin) [inline blocks/img/img.js:6]: I think we can remove this comment. Function names are descriptive enough.

File: blocks/img/img.js [hunk 2]
```diff
@@ -17,4 +86,6 @@ export default function decorate(block) {
       img.style.width = `${width}px`;
     }
   }
+
+  imgZoomable();
```
Comment (thomsebastin) [inline blocks/img/img.js:90]: Is it possible for us to move this to `delayed.js` as this is not required immediately after page load?

**Review by thomsebastin — COMMENTED** [review]
Looks good to me, left a comment.

File: blocks/img/img.js (hunk 2 above)
Comment (adobenitesh) [reply blocks/img/img.js]: Done

File: blocks/img/img.js (hunk 1 above)
Comment (adobenitesh) [reply blocks/img/img.js]: Done

File: blocks/img/img.js (hunk 2 above)
Comment (thomsebastin) [reply blocks/img/img.js]: I think it's better to let the block be loaded as normal. Let's revert the delayed.js logic.

Comment (adobenitesh) [reply blocks/img/img.js:86]: Done

**Review by thomsebastin — APPROVED** [review]

**Pull request merged**